huaweicloud/services/acceptance/secmaster/data_source_huaweicloud_secmaster_alert_rules_test.go

```diff
@@ -0,0 +1,193 @@
+  
+  depends_on = [
+    huaweicloud_secmaster_alert_rule.test1,
+    huaweicloud_secmaster_alert_rule.test2,
+  ]
+}
+
+locals {
+  id          = data.huaweicloud_secmaster_alert_rules.test.alert_rules[0].id
+  name        = data.huaweicloud_secmaster_alert_rules.test.alert_rules[0].name
+  status      = data.huaweicloud_secmaster_alert_rules.test.alert_rules[0].status
+  severity    = data.huaweicloud_secmaster_alert_rules.test.alert_rules[0].severity
+  pipeline_id = data.huaweicloud_secmaster_alert_rules.test.alert_rules[0].pipeline_id
+}
+
+data "huaweicloud_secmaster_alert_rules" "filter_by_id" {
+  workspace_id = "%[2]s"
+  rule_id      = local.id
+}
+
+data "huaweicloud_secmaster_alert_rules" "filter_by_name" {
+  workspace_id = "%[2]s"
+  name         = local.name
+}
+
+data "huaweicloud_secmaster_alert_rules" "filter_by_status" {
+  workspace_id = "%[2]s"
+  status       = [local.status]
+}
+
+data "huaweicloud_secmaster_alert_rules" "filter_by_severity" {
+  workspace_id = "%[2]s"
+  severity     = [local.severity]
+}
+
+data "huaweicloud_secmaster_alert_rules" "filter_by_pipeline_id" {
+  workspace_id = "%[2]s"
+  pipeline_id  = local.pipeline_id
+}
+
+output "is_id_filter_useful" {
+  value = length(data.huaweicloud_secmaster_alert_rules.filter_by_id.alert_rules) > 0 && alltrue(
+    [for v in data.huaweicloud_secmaster_alert_rules.filter_by_id.alert_rules[*].id : v == local.id]
+  )
+}
+
+output "is_name_filter_useful" {
+  value = length(data.huaweicloud_secmaster_alert_rules.filter_by_name.alert_rules) > 0 && alltrue(
+    [for v in data.huaweicloud_secmaster_alert_rules.filter_by_name.alert_rules[*].name : strcontains(v, local.name)]
+  )
+}
+
+output "is_status_filter_useful" {
+  value = length(data.huaweicloud_secmaster_alert_rules.filter_by_status.alert_rules) > 0 && alltrue(
+    [for v in data.huaweicloud_secmaster_alert_rules.filter_by_status.alert_rules[*].status : v == local.status]
+  )
+}
+
+output "is_severity_filter_useful" {
+  value = length(data.huaweicloud_secmaster_alert_rules.filter_by_severity.alert_rules) > 0 && alltrue(
+    [for v in data.huaweicloud_secmaster_alert_rules.filter_by_severity.alert_rules[*].severity : v == local.severity]
+  )
+}
+
+output "is_pipeline_id_filter_useful" {
+  value = length(data.huaweicloud_secmaster_alert_rules.filter_by_pipeline_id.alert_rules) > 0 && alltrue(
+    [for v in data.huaweicloud_secmaster_alert_rules.filter_by_pipeline_id.alert_rules[*].pipeline_id : v == local.pipeline_id]
+  )
+}
+
+`, testAlertRule_buildData(name), acceptance.HW_SECMASTER_WORKSPACE_ID)
+}
+
+func testAlertRule_buildData(name string) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_secmaster_alert_rule" "test1" {
+  workspace_id = "%[1]s"
+  pipeline_id  = "%[2]s"
+  name         = "%[3]s"
+  description  = "this is a test rule created by terraform"
+  status       = "ENABLED"
+  severity     = "TIPS"
+  type = {
+    "name"     = "DNS协议攻击"
+    "category" = "DDoS攻击"
+  }
```

Comment:
better not to use Chinese in the scritpts,
looks there are corresponding English expressions for there values
```suggestion
  type = {
    "name"     = "DNS protocol attacks"
    "category" = "DDoS attacks"
  }
```